test/e2e/antreapolicy_test.go

```diff
@@ -1220,3 +1222,114 @@ func TestAntreaPolicyStatus(t *testing.T) {
 	})
 	assert.NoError(t, err, "Antrea ClusterNetworkPolicy failed to reach expected status")
 }
+
+// TestANPNetworkPolicyStatsWithDropAction tests antreanetworkpolicystats can correctly collect dropped packets stats from ANP if
+// networkpolicystats feature is enabled
+func TestANPNetworkPolicyStatsWithDropAction(t *testing.T) {
+
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+	skipIfAntreaPolicyDisabled(t, data)
+
+	if err := testData.mutateAntreaConfigMap(func(data map[string]string) {
```

Comment:
I think we should disable `NetworkPolicyStats` once we finish this test.

Reply:
I don't think we need to do that, because `mutateAntreaConfigMap` only mutate the string that read from the config file, not the config file itself. The config file will be reread when `setupTest` get called.

Reply:
In `test/e2e/framework.go:1366`, the config is updated actually.
But you are right, each time the `setupTest` gets called the original YAML will be applied.